Dockerfile

```diff
@@ -38,11 +38,15 @@ USER nobody
 WORKDIR /var/www/html
 COPY --chown=nobody src/ /var/www/html/
 
+# Add runit boot scripts
+COPY --chown=nobody config/nginx.run /etc/service/nginx/run
+COPY --chown=nobody config/php.run /etc/service/php/run
+
```

Comment:
I think we should chmod it
`RUN chmod +x /etc/service/nginx/run \
    && chmod +x /etc/service/php/run`

Reply:
The files already have the executable bit, but we can add this line to be secure

Reply:
I don't know why it throw acess denied, when I chmod them, docker run successful

![image](https://user-images.githubusercontent.com/4869137/81098053-0482b380-8f33-11ea-8d18-1a0b33d38826.png)

Reply:
Thank you about amazing pull request <3
![image](https://user-images.githubusercontent.com/4869137/81098141-22e8af00-8f33-11ea-9376-1536071ffe4a.png)

Reply:
Added